gehomesdk/erd/converters/water_heater/erd_waterheater_min_max_temperature.py

```diff
@@ -0,0 +1,14 @@
+from gehomesdk.erd.values import ErdWaterHeaterMinMaxTemperature
+
+from ..abstract import ErdReadOnlyConverter
+from ..primitives import *
+
+
+class ErdWaterHeaterMinMaxTemperatureConverter(
+    ErdReadOnlyConverter[ErdWaterHeaterMinMaxTemperature]
+):
+    def erd_decode(self, value: str) -> ErdWaterHeaterMinMaxTemperature:
```

Comment:
Can you take a look at this: https://github.com/simbaja/gehome/blob/6c1e966f43af3b4029f2c66338a74bd35ad40458/gehomesdk/erd/converters/coffee_maker/erd_ccm_brew_temperature_range_converter.py#L11

Reply:
Got it. just pushed an update to that function. let me know if that looks better.